src/cata_tiles.cpp

```diff
@@ -2345,9 +2345,17 @@ bool cata_tiles::draw_from_id_string( const std::string &id, TILE_CATEGORY categ
         case TILE_CATEGORY::OVERMAP_TERRAIN:
         case TILE_CATEGORY::MAP_EXTRA:
             seed = simple_point_hash( pos );
+            break;
         case TILE_CATEGORY::ITEM:
         case TILE_CATEGORY::TRAP:
         case TILE_CATEGORY::NONE:
+            // graffiti
```

Comment:
Should `ITEM` and `TRAP` be moved below `NONE` to avoid the added check? It looks like graffiti only applies to the `NONE` category

Reply:
Done.